project/Dependencies.scala

```diff
@@ -4,9 +4,10 @@ object Dependencies {
   object Versions {
     val scala212 = "2.12.14"
     val scala213 = "2.13.6"
+    val scala3 = "3.0.1"
 
-    val trace4cats = "0.12.0-RC2"
-    val trace4catsExporterHttp = "0.12.0-RC2"
+    val trace4cats = "0.12.0-RC2+17-d73c7ff3"
```

Comment:
```suggestion
    val trace4cats = "0.12.0-RC2+18-d1292558"
```

Once https://github.com/trace4cats/trace4cats/pull/591 is merged, fixes the `ArithmeticException` issue in CI.